Update/js project status

theecrit opened the pull request:
Closes #107 

##### Changes made:
- Added "Decommissioned projects" and "OpenOakland projects" sections
- Updated project statuses
- Added "Becoming an OpenOakland project" and "Providing feedback" section
- Adjusted project formatting for better readability and info access (moving URL links into project metadata for consistency, etc).

(See commit history for detailed changes)

##### _If your PR changes the appearance of the site, please include desktop and mobile screenshots below_
### Mobile screenshots
<br>

### Desktop screenshots


[projects-screenshot](https://user-images.githubusercontent.com/32755410/98071696-14fbe400-1e19-11eb-872a-5b6dcd28c3aa.png)

theecrit commented:
Well, everything's looking normal in my editor preview and Github's file preview, but my local server is not formatting the markdown properly and I'm not sure what's up there. 
![Screenshot_2020-10-10 Our Projects](https://user-images.githubusercontent.com/32755410/95669457-c7e86300-0b35-11eb-8ee9-54b8a833bd88.jpg)

ckingbailey commented on src/projects.md:
```diff
@@ -1,23 +1,99 @@
 <div id="projects">
-  <h2>Active Projects</h2>
+
+  ## <a id="openoakland-projects"></a>Ongoing projects that support OpenOakland's operation
```
I think I see what you're doing here with an empty <a> tag containing an "id" attribute, which creates a linkable heading. This strikes me as cumbersome to write and confusing to read. Are there other options to do same with plain markdown in Jekyll? If not, I think it should be pure html.

theecrit replied:
I couldn't find any jekyll-specific examples but will look again or swap out.

ckingbailey replied:
I'm wondering if markdown headings in Jekyll get id tags by default, based on this https://jekyllrb.com/docs/configuration/markdown/#kramdown

theecrit replied:
Yes, you're right and I've got those working now, thanks!

ckingbailey commented on src/_data/decommissioned_projects.yml:
```diff
@@ -0,0 +1,10 @@
+-
```
Would be nice to see screenshots of what these new sections look like laid out together

theecrit replied:
Honestly, I'm not happy with the overall layout but this whole section needs a new IA (including moving "how to become a project" onto its own page, probably along with decommissioned/event-based/expired projects).

![projects-screenshot](https://user-images.githubusercontent.com/32755410/97936740-5749f600-1d31-11eb-91f4-35e20b583366.png)

ckingbailey commented:
Thank you for awesome write up of processes and project statuses, and for the maintenance work!

I left a couple of review comments, one of which is request for changes.

TangoYankee commented on src/_sass/wp.scss:
```diff
@@ -1,37 +1,42 @@
-#projects .col1 {
+#projects-openoakland, #projects-active, #projects-inactive, #projects-decommissioned .col1 {
```
For maintainability, it might be easier to instead create a `projects` class and apply it to these headers. 
[Kramdown classes](https://about.gitlab.com/blog/2016/07/19/markdown-kramdown-tips-and-tricks/#applying-classes)

theecrit replied:
I was able to (mostly) resolve this without the need for classes, actually. Thanks for the nudge.

TangoYankee commented on src/projects.md:
```diff
@@ -1,23 +1,102 @@
+
+{% include linkedHeading.html heading="Active projects" level=2 %}
+
+The following projects are official OpenOakland projects that are being actively supported by an existing project team. If you see something that interests you, there are a couple of ways you can get involved:
```
The following ~~projects~~ are official OpenOakland projects that are ~~being~~ actively supported ...

TangoYankee commented on src/projects.md:
```diff
@@ -1,23 +1,102 @@
+
+{% include linkedHeading.html heading="Inactive projects" level=2 %}
+
+The following projects aren’t currently being worked on by a project team and are considered dormant. If you're interested in resuming one of these projects, you'll need to start by updating or completing a new [Project Brief](https://docs.google.com/document/d/1k24P9JiAUEzJLPFRDjVh7aRZexax6NUhfPFLSI3R80M/edit?usp=sharing) (most of these projects were developed prior to our current project develo ...
```
In my experience, terms like `current` and `new` age poorly in documentation. Also, if this is someone's first interaction with OpenOakland, they may want to know we're a helpful group. Finally, repeating `develop` may feel repetitive ;P Perhaps...
>(most of these were created before we implemented our project development process.) Please reach out if you want help in completing the brief and resuming the project.

TangoYankee commented on src/projects.md:
```diff
@@ -1,23 +1,102 @@
+
+Once your final brief is submitted, the Steering Committee will hold a formal vote to approve or decline the project.
+
+### What makes a good project?
```
This might be good to have as the top section. It introduces what we value as organization and could help the reader understand what to expect as they scroll through the lists.

theecrit replied:
I moved it up after "active projects," as it's a little long to go at the top. I actually think it deserves its own page, but I'm going to open a separate issue for that. I'd like to get this info published ASAP.

TangoYankee commented on src/projects.md:
```diff
@@ -1,23 +1,102 @@
+
+{% include linkedHeading.html heading="Becoming an OpenOakland project" level=2 %}
+
+In response to member feedback that has resulted in an ongoing effort to address issues of equity and inclusion in our work, OpenOakland is reevaluating how our projects are vetted, adopted, and developed. This is an ongoing pilot that we continue to iterate on in an effort to ensure that projects serve their intended communities, consider potential unintended consequences, and foster greater inc ...
```
I think this might fit better under '#what makes a good project`. If that section introduces what we're looking for, this `project  brief` section might outline how we plan to hold ourselves accountable to those goals.

TangoYankee commented on src/projects.md:
```diff
@@ -1,23 +1,102 @@
+
+---
+
+{% include linkedHeading.html heading="Providing feedback" level=2 %}
```
This section is still good placed on the bottom of the page. I think it works well as a outro

theecrit replied:
Kept it with the "Becoming..." section, as it's probably more important info than, say decommissioned projects etc.

TangoYankee reviewed (changes requested):
I have a a few recommendations for maintainability and flow. Otherwise, great work! It sets a good tone for how OpenOakland organizes and operates its projects.

theecrit commented:
@ckingbailey @TangoYankee I think I addressed most of your comments, though imperfectly. I've got this mostly working except for what I can figure may be a weird bug with the jekyl anchor headers:
- The links to headers that follow an include that serves a _single_ project jump to the right location on the page.
- The links to headers that follow an include serving _multiple_ projects jump to where the header would be if there were only one project displayed (so, way too high up on the page, as if those projects preceding the header aren’t being recognized and account for).

It’s the weirdest thing and I can’t figure out how to solve for it (or find anything online). Honestly, this whole page needs a real overhaul. Maybe I should just move the "Becoming" and "Feedback" content to their own page linked from this one, and remove the jump links altogether? It's just such a long list. Thoughts?

theecrit commented:
Ugh now I'm seeing the checks failed and no clue what's going on there with the build. Back to the drawing board. 😣

theecrit commented:
Oookaaayyy, thanks to a wonderful fresh set of eyes, I was able to resolve the anchor link issue and this page, while ugly, is now fully functional.
